pkg/dependencies/factory.go

```diff
@@ -297,9 +297,7 @@ func (f *Factory) WithProvider(clusterConfigFile string, clusterConfig *v1alpha1
 				time.Now,
 				skipIpCheck,
 				hardwareConfigFile,
-				skipPowerActions,
```

Comment:
Why is skipPowerActions being removed? I understand that force flag might need cleanup and refactoring. Assuming there is refactoring being done around skipPowerActions as well and that a follow-up PR is imminent